crates/typst-library/src/text/lorem.rs

```diff
@@ -1,13 +1,13 @@
 use crate::foundations::{func, Str};
 
-/// Creates blind text.
+/// ダミーテキストの作成。
 ///
-/// This function yields a Latin-like _Lorem Ipsum_ blind text with the given
-/// number of words. The sequence of words generated by the function is always
-/// the same but randomly chosen. As usual for blind texts, it does not make any
-/// sense. Use it as a placeholder to try layouts.
+/// この関数は与えられた単語数だけラテン語風のダミーテキストである _Lorem Ipsum_ を生成します。
+/// この関数で生成される単語のシーケンスは、常に同一ですがランダムに選ばれます。
+/// 通常のダミーテキストと同様に、意味のないテキストです。
+/// レイアウトを試すプレイスホルダーとして使用してください。
```

Comment:
```suggestion
/// レイアウトを試すプレースホルダーとして使用してください。
```
細かい点になってしまいますが「プレースホルダー」表記のほうが一般的な気がします。

Reply:
そうですね、採用させてもらいました。ありがとうございます。